src/components/Checkbox/stories/Checkbox.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import Checkbox from "..";
+
+export default {
+  title: "Components/Checkbox",
+  component: Checkbox,
+};
+
+export const Default = () => {
+  return <Checkbox />;
```

Comment:
Once you made the `onChange` update, this story should also make sure that you can use the new `checked` and `onChange` props